ddtrace/profiling/collector/_traceback.pyx

```diff
@@ -67,22 +28,16 @@ cpdef traceback_to_frames(traceback, max_nframes):
     :return: The serialized frames and the number of frames present in the original traceback.
     """
     tb = traceback
-    frames = []
-    nframes = 0
+
+    cdef list frames = []
```

Comment:
`cdef tb` too ? 🤷🏻

Reply:
Yep we could do, but I wouldn't bother too much as this is not a mile off from what we currently have implemented, so we might as well just close this PR entirely as I don't think we gain much from these little tweaks for now